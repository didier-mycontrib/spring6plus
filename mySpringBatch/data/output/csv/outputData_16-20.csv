firstname;lastname;age;active
prenom16;nom16;33;true
prenom17;nom17;33;true
prenom18;nom18;33;true
prenom19;nom19;33;true
prenom20;nom20;33;true
