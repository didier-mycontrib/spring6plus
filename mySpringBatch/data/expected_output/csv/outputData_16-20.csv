id;firstname;lastname;age;active
16;prenom16;nom16;33;true
17;prenom17;nom17;33;true
18;prenom18;nom18;33;true
19;prenom19;nom19;33;true
20;prenom20;nom20;33;true
